backend/ml/mehestan/global_scores.py

```diff
@@ -294,23 +297,19 @@ def compute_scaled_scores(
     df["delta_s"].fillna(0, inplace=True)
     df["delta_tau"].fillna(0, inplace=True)
     df["uncertainty"] = (
-        df["s"] * df["uncertainty"]
-        + df["delta_s"] * df["score"].abs()
+        df["s"] * df["raw_uncertainty"]
+        + df["delta_s"] * df["raw_score"].abs()
         + df["delta_tau"]
     )
-    df["score"] = df["score"] * df["s"] + df["tau"]
+    df["score"] = df["raw_score"] * df["s"] + df["tau"]
     df.drop(["s", "tau", "delta_s", "delta_tau"], axis=1, inplace=True)
```

Comment:
I think the difference between the 2 branches can be explained here:
when applying the scaling here, "s" and "tau" contain the scaling for non-supertrusted users only. 
It was not a problem with the previous implementation: the scaling for supertrusted users was already applied on line 271-275, and this second step was a noop for them.

Now that "score" and "uncertainty" are re-computed based on "raw_scores" values, the scaled scores for supertrusted users are lost. I think that the fix could simply be to join with "all_scalings" in this second step, instead of using non_supertrusted_scaling only.

Reply:
Thanks for the eagle eyes on that issue :+1: 

If I rephrase correctly as far that I understand : the second step is only for the non supertrusted, as we use the (s,tau) =(1,0) trick to keep the previous computation on supertrusted users. Therefore, I reuse the "is_supertrusted" attribute to filter the rows to select non supertrusted and to apply the right computation only for that targeted population.

On dev-env, it seems like that we don't have non supertrusted user, I can't test properly. On working days, can you do a mehestan run with this PR and check if the global scores and the sigmoid_scales don't change from current values, with proper production-like data  ?

I think I am good on my side